deploy/k8s/charts/trustification/values.yaml

```diff
@@ -212,4 +213,13 @@ modules:
     image: {}
     rust: {}
 
+  report:
+    enabled: true
```

Comment:
I would keep report disabled by default, especially if we can't enable easy authentication.